dirsrvtests/tests/suites/filter/filter_test.py

```diff
@@ -373,15 +373,16 @@ def test_match_large_valueset(topology_st):
             config_ldbm = DSLdapObject(inst, DN_CONFIG_LDBM)
             config_ldbm.set('nsslapd-cache-autosize', '0')
         be_groups.replace('nsslapd-cachememsize', groups_entrycache)
-    except:
-        raise
+    except Exception as e:
+        raise e
```

Comment:
I think this part is completely redundant...
It basically just catches and immediately re-raises the same exception without performing any additional actions.

Even though it's better than bare `except: raise` - still, I'd just remove it:)

Reply:
I was just fixing lint errors, I can take it out.